Flow.Launcher/ResultListBox.xaml.cs

```diff
@@ -137,6 +137,8 @@ private void ResultList_MouseMove(object sender, MouseEventArgs e)
 
             isDragging = false;
 
+            App.API.HideMainWindow();
+
```

Comment:
Main window will show later when finishing dragging since we're calling `ChangeQuery()` in line 156.

Reply:
Main window will only show later when it's a Move action now. Should we amend this? @onesounds @jjw24

Reply:
Sorry what do you mean? Does change query call make the search window visible again? When I was testing this with moving a file the search window wasn't showing after the drop was done and had to retrigger window to appear.

Reply:
When I was testing this PR something did bring back the search window, and I thought it was ` ChangeQuery`. But now I am confused about the logic. Will test later this week.